CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 - DDFileLogger log message is overridden #924
 - Fix thread safety issue in DDFileLogger #986
 - Fix availability checks and memory leak #996
+- Added `initialFileContents` property to `DDLogFileManagerDefault` to set log file header
```

Comment:
One last left here and we're good to merge.

Reply:
Oh, right. Forgot that one, thanks :-)
I'm also implementing some test cases so both OSSPolice and codecov will hopefully be happy.